trikot-viewmodels-declarative-flow/compose-flow/src/main/kotlin/com/mirego/trikot/viewmodels/declarative/compose/viewmodel/VMDImage.kt

```diff
@@ -45,32 +52,6 @@ import kotlinx.coroutines.flow.mapNotNull
 private const val TAG = "VMDImage"
 private const val MAX_BITMAP_SIZE = 100 * 1024 * 1024 // 100 MB, taken from android.graphics.RecordingCanvas
 
-@Composable
-fun VMDImage(
```

Comment:
why remove this one?  This is the most performant option and should be used 99% of the times.